Notifications indicator

Tunous opened the pull request:
![screenshot_1495615866](https://cloud.githubusercontent.com/assets/5156340/26410863/b7e25d12-40a4-11e7-8f72-cdb9bd0e5ade.png)

Closes #599.

Tunous commented:
Instead of the icon could also display a number of notifications but I personally like how that little dot looks.

maniac103 commented on app/src/main/java/com/gh4a/BaseActivity.java:
```diff
@@ -96,6 +100,10 @@
+    private ImageView mNotificationsIndicator;
+
+    private Drawable mTintedCircleIcon;
+    private Drawable mCircleIcon;
```
My gut feeling is that all of the changes in this file should be in HomeActivity. None of this is relevant to any other activity.

maniac103 commented:
Wouldn't it be more obvious to just use a different icon instead of the dot? I agree it looks nice, but it's not exactly intuitive.

sharkykh commented:
> Wouldn't it be more obvious to just use a different icon instead of the dot? I agree it looks nice, but it's not exactly intuitive.

" <B>NEW</B> "?

Tunous commented:
![screenshot_1495711084](https://cloud.githubusercontent.com/assets/5156340/26448260/a03fd2ea-414c-11e7-9599-648015f9c347.png)

What about this?

maniac103 commented:
Yeah, I like the dot better than this as well. Just to be clear, by 'different icon' I meant using a different icon on the left side of the text when there are present notifications, not replacing the dot with something else.

Tunous commented:
![screenshot_1495881548](https://cloud.githubusercontent.com/assets/5156340/26520796/a405cf4a-42d9-11e7-9f8a-6fcf3af01b86.png)

This does look nice but it won't be easy to notice it unless you know where to look.
What about combining this icon with the dot? Dot will draw user attention and changed icon will make it easier to see what the dot means.

sharkykh commented:
> What about combining this icon with the dot? Dot will draw user attention and changed icon will make it easier to see what the dot means.

@Tunous Like on Github. A bell icon all the time, a bell with a blue dot when there are unread notifications.
I think I suggested this in #599 as well.

Tunous commented:
I don't think it's possible to use colored icons in this drawer implementation... Notice how it automatically changes text and icon to green. Similarly, it colors everything to that gray color when not selected.

sharkykh commented:
@Tunous 
Oh, that's too bad...
If that's the case - a dot to the right of the drawer item is better, IMO.

Tunous commented:
Now in addition to the dot, the icon on the left side is also changed. Also moved the code as suggested.

![screenshot_1496488644](https://cloud.githubusercontent.com/assets/5156340/26753095/03272c6e-485f-11e7-90e6-d506fa0aef67.png)

maniac103 commented:
Looks great. Just had some minor comments.

Tunous commented:
Updated.
